splinterd/Cargo.toml

```diff
@@ -43,6 +44,7 @@ tempdir = "0.3"
 
 [features]
 default = []
+health-service-enabled = [ "health" ]
```

Comment:
this feature should probably be just `health-service` (as enabled state is determined by if the feature is included in the build)

Reply:
Also, move it below experimental below and add to the experimental block.